gui/Graph.cpp

```diff
@@ -93,6 +99,24 @@ void Graph::drawCircle(Circle *circle) {
 		SDL_RenderDrawPoint(this->renderer, centerX + y, centerY + x);
 		SDL_RenderDrawPoint(this->renderer, centerX - y, centerY - x);
 		SDL_RenderDrawPoint(this->renderer, centerX - y, centerY + x);
+		if (filled) {
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX + x, centerY - y);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX + x, centerY + y);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX - x, centerY - y);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX - x, centerY + y);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX + y, centerY - x);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX + y, centerY + x);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX - y, centerY - x);
+			SDL_RenderDrawLine(
+				this->renderer, centerX, centerY, centerX - y, centerY + x);
```

Comment:
I think this can be optimised
Couldn't you draw less lines by drawing a line from (center + x, center + y) to (center - x, center - y), etc?